azurerm/resource_arm_virtual_machine_scale_set_test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
+const scaleSetResourceName = "azurerm_virtual_machine_scale_set.test"
```

Comment:
I understand the motive but as this ends up being a global constant we tend not to do this. Could we please revert this and use the existing pattern for consistency?

Reply:
Hi Katy, this was updated per your request. You wanted to move the strings out of the tests into a single var. Please comment on which you are favoring, duplication or const var.